azext_edge/tests/generators.py

```diff
@@ -23,3 +25,8 @@ def generate_names(prefix: str = "", count: int = 1, max_length: int = 48) -> Un
 
 def get_zeroed_subscription() -> str:
     return "00000000-0000-0000-0000-000000000000"
+
+
+def generate_random_string(size: int):
```

Comment:
is the generate_id no longer enough?

Reply:
generate_id produces a uuid str to convey an Id like value with fixed size.  

generate_random_string is meant to produce a string of chosen length with wider range of characters (which can be further customized)

generate_id could be refactored to have variable length but then the logic would have to concatenate chunks and trim the extra + be more limited in character range.